### DISCUSSION TITLE
The error message indicating that the item is out of stock frequently disappears on the product page.

### DISCUSSION THREAD

**Discussion by nilaykumardeveloper** [breezefront/community#160] [1. Help]
Hi @vovayatsyuk,

On the product detail page (PDP), when we add an out-of-stock size variation, an error message is initially displayed, but it disappears within seconds. The issue described works properly on the Magento Luma theme.
See video https://www.loom.com/share/7818a766664648ae9bff4bd8267c68f2?sid=ee2f8986-064a-4d34-88aa-25cff23533e0

Kindly assist with it.
Thanks

**Comment by vovayatsyuk** [comment]
I cannot reproduce this issue. The message does not disappear on my setup.

**Comment by vovayatsyuk** [comment]
Maybe you are using some Ajax cart module? Could you share a link to your site?

**Reply by nilaykumardeveloper** [reply]
Yes, It was in some Ajax request and now resolve it at our end.
Thank you.

**Reply by parthvisavadiya** [reply]
> Yes, It was in some Ajax request and now resolve it at our end. Thank you.

Can you let me know, what was the issue and what you have fixed?
